### PR TITLE
feat(origin-ui-core): Add deposit action

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

File: packages/localization/src/languages/pl.json [hunk 1]
```diff
@@ -78,7 +78,12 @@
                 "returnToInbox": "Zwróć do skrzynki",
                 "buy": "Kup",
                 "request": "Żądaj",
-                "withdraw": "Wycofać"
+                "withdraw": "Wycofać",
```
Comment (kosecki123) [inline packages/localization/src/languages/pl.json]: Please use “Wypłać”

File: packages/localization/src/languages/pl.json [hunk 2]
```diff
@@ -78,7 +78,12 @@
...
+                "deposit": "Kaucja"
```
Comment (kosecki123) [inline packages/localization/src/languages/pl.json]: Please use “Wpłać”

File: packages/origin-ui-core/src/features/certificates/sagas.ts [hunk 3]
```diff
@@ -288,6 +288,63 @@ function* requestPublishForSaleSaga(): SagaIterator {
...
 }
 
+function* requestDepositSaga(): SagaIterator {
+    while (true) {
+        const action: IRequestPublishForSaleAction = yield take(
+            CertificatesActions.requestDepositCertificate
+        );
+        const exchangeClient: IExchangeClient = yield select(getExchangeClient);
+
+        if (!exchangeClient.getAllTransfers) {
```
Comment (kosecki123) [inline packages/origin-ui-core/src/features/certificates/sagas.ts]: Wondering about this check. Why we need that?

File: packages/origin-ui-core/src/features/certificates/sagas.ts [hunk 4]
```diff
@@ -288,6 +288,63 @@ function* requestPublishForSaleSaga(): SagaIterator {
...
+
+        yield put(setLoading(true));
+
+        try {
+            const account: ExchangeAccount = yield call([
+                exchangeClient,
+                exchangeClient.getAccount
+            ]);
+
+            if (source === CertificateSource.Blockchain) {
```
Comment (kosecki123) [inline packages/origin-ui-core/src/features/certificates/sagas.ts]: I guess we don’t need that check here, or if we need then move it up as a guard closure:

If (source !== ...)...

**Review by kosecki123 — APPROVED** [review]
LGTM

File: packages/origin-ui-core/src/features/certificates/sagas.ts (hunk 3 above)
Comment (JGiter) [reply packages/origin-ui-core/src/features/certificates/sagas.ts]: It left after my experiments with certificates. Deleted

File: packages/origin-ui-core/src/features/certificates/sagas.ts (hunk 4 above)
Comment (JGiter) [reply packages/origin-ui-core/src/features/certificates/sagas.ts]: Fixed

**Pull request merged**